apps/examples/sensorbd_demo/examples/spi_k6ds3.c

```diff
@@ -163,9 +163,8 @@ void k6ds3_main(int argc, char *argv[])
 		gy = (gyl | (gyh << 8));
 		gz = (gzl | (gzh << 8));
 
-		printf("ACCEL:x(0x%04X) y(0x%04X) z(0x%04X) GYRO:x(0x%04X) y(0x%04X) z(0x%04X)\n",
-			ax, ay, az, gx, gy, gz);
+		printf("ACCEL:x(0x%04X) y(0x%04X) z(0x%04X) GYRO:x(0x%04X) y(0x%04X) z(0x%04X)\n", ax, ay, az, gx, gy, gz);
```

Comment:
This is the Tizen coding style I sent before. If you want to keep original codes, you can use \ at end of line.